presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -116,7 +127,7 @@ std::shared_ptr<connector::ColumnHandle> toColumnHandle(
         hiveColumn->name,
         toHiveColumnType(hiveColumn->columnType),
         stringToType(hiveColumn->typeSignature),
-        stringToType(hiveColumn->typeSignature),
+        hiveTypeParser.parse(hiveColumn->hiveType),
```

Comment:
Should we add a check that Hive type matches Presto type or do we have it somewhere else?

Reply:
We have the check in HiveColumnHandle ctor that added in Velox side. thanks!